src/components/Feed.tsx

```diff
@@ -1,16 +1,59 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Tweetbox from "./Tweetbox";
 import "./Feed.css";
 import Post from "./Post";
+import db from "../firebase";
+
+interface PostData {
+  avatar: string;
+  displayName: string;
+  image: string;
+  text: string;
+  userName: string;
+  verified: boolean;
+}
+
+const Feed: React.FC = (): JSX.Element => {
```

Comment:
If you have used React.FC, you don't need to use JSX.Element

Reply:
Noted, correcting the issue.